### PR TITLE
Assembler: Another round of tweaks

### PR DESCRIPTION
**Pull request by richtabor**
- Adjust button sizes
- Better line-height for headings
- Experiment for improving title-less page.html template display, when not using patterns.
- Add 404.html
- Update archive, single post templates.

### CODE REVIEW

File: assembler/templates/search.html [hunk 1]
```diff
@@ -1,57 +1,55 @@
...
-<!-- /wp:group -->
+<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
+<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
 
-<!-- wp:post-featured-image {"isLink":true,"aspectRatio":"3/2"} /-->
+<!-- wp:query {"queryId":0,"query":{"perPage":10,"pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-query alignwide"><!-- wp:query-no-results -->
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">No search results.</p>
```
Comment (alaczek) [inline assembler/templates/search.html]: We should make this into a pattern to reuse across templates, and wrap this text in translation function.

File: assembler/templates/archive.html [hunk 2]
```diff
@@ -1,40 +1,49 @@
...
+<!-- wp:query-title {"type":"archive","textAlign":"center","showPrefix":false,"align":"wide","style":{"typography":{"lineHeight":"1"}},"fontSize":"large"} /-->
+
+<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
+<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:query {"queryId":0,"query":{"perPage":10,"pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-query alignwide"><!-- wp:query-no-results -->
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">No search results.</p>
```
Comment (alaczek) [inline assembler/templates/archive.html]: We should make this into a pattern to reuse across templates, and wrap this text in translation function.

File: assembler/templates/404.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!-- wp:template-part {"slug":"header"} /-->
+
+<!-- wp:group {"tagName":"main","metadata":{"name":"Main"},"layout":{"type":"constrained","contentSize":"400px","wideSize":"720px"}} -->
+<main class="wp-block-group"><!-- wp:spacer {"height":"var:preset|spacing|70"} -->
+<div style="height:var(--wp--preset--spacing--70)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:heading {"textAlign":"center","align":"wide"} -->
+<h2 class="wp-block-heading alignwide has-text-align-center" id="page-not-found">Page Not Found</h2>
```
Comment (alaczek) [inline assembler/templates/404.html]: We should make this into a pattern and wrap this text in translation function.

**Review by alaczek — CHANGES_REQUESTED** [review]

**Comment by alaczek** [comment]
I reviewed the templates and have a few questions and comments:
1. Was there a specific reason to turn page with title into a pattern? 
2. Should we have some sort of post meta (date and author or category) in the single post template?
3. The post titles in archives and search templates look off:
 * some are centered, some are not
 * the line height is very tight
 * the font size is different than in index.html (which also has archive title - I think that's a bug?)
image

**Comment by richtabor** [comment]
Very strange about the centered titles. I'll take a look today.

**Comment by alaczek** [comment]
This bit in theme.json forces the title to be centered:
```
"core/post-title": {
                "css": ":not([class*=\"text-align-\"]){text-align:center;}",
                "typography": {
                    "fontSize": "var(--wp--preset--font-size--x-large)"
                }
            },
```

**Comment by richtabor** [comment]
I made all the requested changes; re-requesting a review + deployment. Thanks!

> Should we have some sort of post meta (date and author or category) in the single post template?

Let's revisit the blog design once the theme/assembler work is being tested and in flight.

**Comment by alaczek** [comment]
> Let's revisit the blog design once the theme/assembler work is being tested and in flight.

Yep, agreed.

**Pull request merged**